feature/02_create_account.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Create an account
+
+    As a common user
+    I want to Create an account
+    In order to have an user account
+
+    Scenario: Create an account
+        Given the user is on demoblaze page
+        When the user complete the Sign Up form
```

Comment:
2 things:
- Third person verbs end with a s
`completes` instead of `complete`
- instead of saying the user completes the signup form, you should be naming the business rule `the user creates an account` that implies that you do all the actions, not only complete the form (open the form, complete the form, submit the form)